### ISSUE TITLE
Train ARIMA, XGBoost and LSTM models per product

### ISSUE THREAD

**Issue by MARITZ-VJ** [BAnalysis-3-2/StockWise-AI#36]
-Fit a time series model using historical demand for each product.
-Use sequential features to train a neural network that captures temporal dependencies.
-Use engineered features to train a gradient boosting model for each product.
